dipy/tracking/tests/test_streamline.py

```diff
@@ -354,7 +353,6 @@ def test_set_number_of_points_memory_leaks():
         streamlines.append(rng.randn(rng.randint(10, 100), 3).astype(dtype))
 
     list_refcount_before = get_type_refcount()["list"]
-    rstreamlines = set_number_of_points(streamlines, nb_points=2)
```

Comment:
Why do you remove this ?

Reply:
Unused variable, declared variable without details and useless afterward